ui/apps/platform/src/services/types.ts

```diff
@@ -1,5 +1,13 @@
 import { ApiSortOption } from 'types/search';
 
+/* The type for pagination data stored and generated client side */
+export type ClientPagination = {
+    page: number;
+    perPage: number;
+    sortOption?: ApiSortOption;
+};
+
```

Comment:
Interestingly enough, we don't have a type defined for this common pattern. I'll follow up with a refactor of Node/Platform CVEs to incorporate this change.